app/views/dashboard/firms/_form.html.erb

```diff
@@ -0,0 +1,59 @@
+<% if @firm.errors.any? %>
+  <div class="t-validation-summary">
+    <%= f.validation_summary %>
+  </div>
+<% end %>
+
+<section class="form-section">
+  <div class="form-section__heading-with-button">
+    <% if f.object.trading_name? %>
+      <h2 class="form-section__heading-with-button__heading"><%= t('dashboard.firm_edit.trading_name_heading') %></h2>
+    <% else %>
+      <h2 class="form-section__heading-with-button__heading"><%= t('dashboard.firm_edit.firm_heading') %></h2>
+      <a class="form-section__heading-with-button__button button button--primary"><%= t('dashboard.add_trading_name_button') %></a>
```

Comment:
https://github.com/moneyadviceservice/rad/commit/b9dc8cc6cae531400a19e0fa2e24df25354660a3

Focused on `.form-section__heading-with-button*` as it seemed the only really egregious example.